Q1-2023 community board update (#1906)
writethedocs/www: docs/blog/2023-Q1-community-board.rst

.. post:: Mar 7, 2023
   :tags: community board, news, report
   :author: Mikey Ariel

Write the Docs Community Board Quarterly Update Q1-2023
=========================================================

Greetings, documentarians! Mikey Ariel here, chairperson of the newly formed community board and former head organizer of the Prague conference. 

As you might have heard, at the end of 2022 I stepped down from my role as conference chair and transitioned to the new role of community board chair. The board meets quarterly during the first month of the quarter, and is comprised of each team lead in the WTD organization. 

> We hope that this group will be able to discuss and drive more long-term strategic initiatives and collaboration for the community. 

Reviewer: Maybe make this more concrete than 'hope'?

> We hope that this group will be able to discuss and drive more long-term strategic initiatives and collaboration for the community. 

Author: ```suggestion
Our main goal with this group is to discuss and drive more long-term strategic initiatives and collaboration for the community. 
```

In the meantime, I can share with you that we have some great ideas from the Meetups team on how to improve our social media strategy, and we'd love to get some more from the community. Stay tuned for more details in one of the upcoming newsletters.

We will also start reviewing the global community initiative backlog and triage as many of the requests that have been accumulating as we can. I am excited to be able to curate all the lovely ideas and suggestions that we've received from our teams and from community members, so that we can focus our efforts on what matters most to the community (that's all of you!).

What is the community board
---------------------------

In case you want a refresher on how the board was formed and who is a member, here is the TL;DR:

The `Community Board <https://www.writethedocs.org/team/#community-board>`_ exists to curate the overall strategic direction of the community.

The members are the team leads of each of the Write the Docs team, as well as the conference chairs of our various conferences:

* Mikey Ariel (CoreOps, Brand and design, board chair)
* Eric Holscher (CoreOps, Portland chair)
* Samuel Wright (CoreOps, Infrastructure)

> * Sasha Romijn (CoreOps, Support)

Reviewer: ```suggestion
* Sasha Romijn (CoreOps, Support, Atlantic Chair)
```
